tcms/testruns/static/testruns/js/get.js

```diff
@@ -250,28 +268,29 @@ function addLinkToExecution(event, testExecution) {
         const isDefect = $('.add-hyperlink-form #defectCheckbox').is(':checked')
         const updateTracker = true
 
-        jsonRPC('TestExecution.add_link', [{
-            execution_id: testExecution.id,
-            url: url,
-            name: name,
-            is_defect: isDefect,
-        }, updateTracker], link => {
-            const testExecutionRow = $(event.target).closest(`.test-execution-${testExecution.id}`);
-            animate(testExecutionRow, () => {
-                const ul = $(`.test-execution-${testExecution.id} .test-execution-hyperlinks`)
-                ul.append(renderLink(link))
+        testExecutionIDs.forEach(testExecutionId => {
+            jsonRPC('TestExecution.add_link', [{
+                execution_id: testExecutionId,
+                url: url,
+                name: name,
+                is_defect: isDefect,
+            }, updateTracker], link => {
+                const testExecutionRow = $(`div.list-group-item.test-execution-${testExecutionId}`);
+                animate(testExecutionRow, () => {
+                    const ul = $(`.test-execution-${testExecutionId} .test-execution-hyperlinks`)
```

Comment:
Selector can be simplified b/c this is a child of testExecutionRow. You can search directly under the current row.